backend/de.metas.cucumber/src/test/java/de/metas/cucumber/stepdefs/C_BPartner_StepDef.java

```diff
@@ -227,7 +228,9 @@ private void createC_BPartner(@NonNull final Map<String, String> tableRow, final
 			bPartnerRecord.setInvoiceRule(invoiceRule);
 		}
 
-		bPartnerRecord.setDeliveryRule(DELIVERYRULE_Force);
+		final String deliveryRule = DataTableUtil.extractStringOrNullForColumnName(tableRow, "OPT." + COLUMNNAME_DeliveryRule);
+		bPartnerRecord.setDeliveryRule(Check.isNotBlank(deliveryRule) ? deliveryRule : DELIVERYRULE_Force);
```

Comment:
here you could also use `de.metas.common.util.CoalesceUtil#firstNotBlank(java.lang.String...)` ..just as hint for next time :-)